cmd/daprd/components/crypto_jwks.go

```diff
@@ -21,5 +21,6 @@ import (
 )
 
 func init() {
-	cryptoLoader.DefaultRegistry.RegisterComponent(jwks.NewJWKSCrypto, "dapr.jwks")
+	// TODO: in future rm the dapr prefix
+	cryptoLoader.DefaultRegistry.RegisterComponent(jwks.NewJWKSCrypto, "jwks", "dapr.jwks")
```

Comment:
Please revert all instances of adding non-`dapr` prefixes. These groups are intentional, and describe components whose provider is Dapr itself, rather than some other external provider (Azure, AWS etc.)